exo-setup/src/docker-setup.ls

```diff
@@ -32,17 +32,19 @@ class DockerSetup
       links: @_get-docker-links!
       environment: @_get-docker-env-vars!
       depends_on: @_get-service-dependencies!
-    for dependency, dependency-config of @service-config.dependencies
-      docker-config[dependency + dependency-config.dev.version] = @_get-service-dependency-docker-config dependency, dependency-config.dev
+    if @service-config.dependencies
```

Comment:
This is an array so should be `for dependency in @service-config.dependencies or []`. I think that works.